src/UI/Dependency/Dummy/DummyFeatureManager.ts

```diff
@@ -40,4 +40,7 @@ export class DummyFeatureManager implements FeatureManager {
   getLicenseInformation(): StatusLicense {
     throw new Error("Method not implemented.");
   }
+  setServerStatus(_serverStatus: ServerStatus): void {
```

Comment:
why is serverStatus part of the featureManager?

Reply:
all the features are from the server status, previously it was accessed through store, which is not longer the case so we need to set it there

Reply:
hmmm, why do we need to set the entire serverStatus in the featureManager? It doesn't make sense from a logical point of view, to have the full serverStatus under a featureManager. It's not coherent or predictable to be here.

Reply:
how else we want to access features that are stored in the serverStatus? I can extract those that are used, but so far  the logic of the app was that app fetched server status, then at first time when some data from FeatureManager is accessed manager tries to get servers status, if there is none then it crashes the app with error. 

I think we can refactor FeatureManager to be hook on it's own to fetch the status and store it's features but that's beyond the scope and I can add it to the queue

Reply:
https://github.com/inmanta/web-console/issues/6331